crypto_commons/generic.py

```diff
@@ -34,6 +37,8 @@ def long_to_bytes(data):
     :param data: integer
     :return: ascii encoded string
     """
+    if data == 0:
+        return '\0'
```

Comment:
```suggestion
        return "\0"
```

It seems we're mostly using doublequotes in other places